cmsroles/admin.py

```diff
@@ -70,6 +70,11 @@ def has_change_permission(self, request, obj=None):
         user = request.user
         return is_site_admin(user) and len(get_administered_sites(user)) > 0
 
+    def change_view(self, request, object_id=None, *args, **kwargs):
+        if object_id:
```

Comment:
object_id naming is confusing... what is the connection between object_id and permission?